.github/workflows/e2e.yml

```diff
@@ -82,20 +82,18 @@ jobs:
           name: Debug files
           path: ./**/target/e2e-debug
 
-      - name: Deploy report
-        uses: peaceiris/actions-gh-pages@v3
-        if: always()
-        with:
-          github_token: ${{ secrets.AUTH_TOKEN }}
-          publish_dir: ./plugin/target/cucumber-reports
-          destination_dir: cucumber-reports/${{ github.ref }}
-
       - name: github-status-action
         uses: Sibz/github-status-action@67af1f4042a5a790681aad83c44008ca6cfab83d
         if: always()
         with:
           authToken: ${{ secrets.AUTH_TOKEN }}
           state: success
           context: Cucumber report
-          target_url: https://data-integrations.github.io/servicenow-plugins/cucumber-reports/${{ github.ref }}/advanced-reports/cucumber-html-reports/overview-features.html
           sha: ${{github.event.pull_request.head.sha || github.sha}}
+
+      - name: Upload reports to GCS
```

Comment:
It should be the second last step not the last.

Reply:
changed to second last step.